helpers/milestones.js

```diff
@@ -24,5 +24,5 @@
  * @todo Add description for the namespace and the properties
  */
 module.exports = {
-	disableDappTransfers: 0,
+	disableDappTransfers: 5594491,
```

Comment:
Based on the condition in code, and the migration announcement we made this seems correct. 

https://github.com/LiskHQ/lisk/blob/e3f3b595a78ff6c48bd85562292b44a8efa7863e/logic/out_transfer.js#L83-L85